jsk_panda_robot/panda_eus/models/dual_panda2.urdf

```diff
@@ -0,0 +1,1260 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
if this is autogenerated file, we should remove this.

Reply:
Sorry, I'll remove this.

Reply:
Done